applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -405,51 +405,52 @@ The test command is not supported.
 Native TLS CMNG #XCMNG
 ======================
 
-The ``#XCMNG`` command manages the credentials to support :ref:`CONFIG_SLM_NATIVE_TLS <CONFIG_SLM_NATIVE_TLS>`.
-This command is similar to the modem ``%CMNG`` command.
+The ``#XCMNG`` command manages the credentials to support :ref:`CONFIG_SLM_NATIVE_TLS <CONFIG_SLM_NATIVE_TLS>`, which is activated with the :file:`overlay-native_tls.conf` configuration file.
+This command is similar to the modem ``%CMNG`` command, but it utilizes Zephyr setting storage instead of modem credential storage.
+
+.. note::
+
+   The Zephyr setting storage is accessible through the debug port of the nRF91 Series devices.
 
 Set command
 -----------
 
 The set command is used for credential storage management.
-The command writes, reads, deletes, and checks the existence of keys and certificates.
+The command writes and deletes credentials.
+It can also list the ``sec_tag`` and ``type`` values of existing credentials.
 
 Syntax
 ~~~~~~
 
-The following is the syntax when :ref:`CONFIG_SLM_NATIVE_TLS <CONFIG_SLM_NATIVE_TLS>` is selected:
 ::
 
-   #XCMNG=<opcode>[,<sec_tag>[,<type>[,<content>]]]
+   #XCMNG=<op>[,<sec_tag>[,<type>[,<content>]]]
 
-The ``<opcode>`` parameter is an integer.
-It accepts the following values:
+The ``<op>`` parameter can have the following integer values:
 
 * ``0`` - Write a credential.
-* ``1`` - List credentials (currently not supported).
-* ``2`` - Read a credential (currently not supported).
+* ``1`` - List credentials.
 * ``3`` - Delete a credential.
 
-The ``<sec_tag>`` parameter is an integer ranging between ``0`` and ``2147483647``.
-It is mandatory for *write*, *read*, and *delete* operations.
-It is optional for *list* operations.
+The ``<sec_tag>`` parameter can have an integer value ranging between ``0`` and ``2147483647``.
```

Comment:
```suggestion
The ``<sec_tag>`` parameter can have integer values ranging between ``0`` and ``2147483647``.
```

Reply:
This isn't correct. Plural would be used if we listed the possible values, but singular is used when noting a range.